index.html

```diff
@@ -90,20 +92,102 @@ <h3>
                     <a href="#">See Project</a>
                 </div>
             </div>
-        </div>       
+        </div>
+        <div class="project-2">
+            <div class="first_project_img_div">
+            <img src="img/default_project_img.JPG" alt="first project image" class="first_project_image">
+            </div>
+            <div class="first-project">
+              <h3>
+                Setup-Portfolio-Mobile-1st <br>
```

Comment:
- [ ] The projects section should be center , Kindly center this section to stick to the Figma design and come out with a beautiful design 👍 

![Capture 3](https://user-images.githubusercontent.com/80458199/228806380-559c6e85-7f06-47a1-8544-c1e018ef4236.PNG)